kong-admin-api.yml

```diff
@@ -1,189 +1,125 @@
+servers:
```

Comment:
I'm worried this seems like a big diff

Reply:
This is caused because libyaml does not support "sorting objects before exporting". It iterates over Lua tables with `pairs`/`next`. Our "root" is a yaml object, not an array, so this means that even at the top-level things are not guaranteed to remain on the same order after every run of the command.